packages/@misk/simpleredux/src/reducer.ts

```diff
@@ -16,7 +16,7 @@ const initialState = defaultRootState(simpleTag)
  * Merges dispatched action objects on to the existing (or initial) state to generate new state
  */
 export function SimpleReduxReducer(
-  state = initialState,
+  state: any = initialState,
```

Comment:
This change is because the type of `initialState` changed (from any -> an Immutable map explicitly). This caused the `reducer.d.ts` file not to be generated, making the bundle test fail

https://github.com/cashapp/misk-web/blob/1ffe3587f76941180e562ab69a15ebb7b3678b65/packages/%40misk/simpleredux/tests/bundle.test.ts#L19-L25